packages/eslint-config/index.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
     node: true,
   },
   rules: {
+    curly: ['error', 'multi'],
```

Comment:
Why not use `all`?

One of the main points we use and fail with is `if` where we do a one liner and then add another line and then we have to add the brackets. I think just add brackets always and that's it. 

Also minimizes changes made since brackets will be there from the beginning.

Reply:
> Good point. Stricter sometimes is better. We should maybe open a poll in the web team channel. Wdyt?

@sshanzel yes! 👌 , we can also have a quick discussion on the next web team meet to quickly go through pros/cons I think it should be an easy choice.